listenbrainz_spark/recommendations/utils.py

```diff
@@ -9,3 +15,40 @@ def save_html(filename, context, template):
     with open(outputfile, 'w') as f:
         html = template_environment.get_template(template).render(context)
         f.write(html)
+
+def initialize_spark_session(app_name):
```

Comment:
Instead of creating a new function, just make these changes in the `init_spark_session` function in  listenbrainz_spark/__init__.py

Reply:
Oh and raise the exception instead of returning bool